### PR TITLE
Add Ghanaian regions to populate the State field on checkout

### PR DESCRIPTION
**Pull request by tolu-paystack**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds the 16 regions of Ghana to states.php to populate the State field on the billing/shipping form

### How to test the changes in this Pull Request:

1. Select Ghana when filling out billing/shipping information on an order
2. Currently the State field is a regular text input. This change will make it a dropdown with the 16 regions as options

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
Adds Ghanaian regions to populate the State dropdown on billing/shipping form

### CODE REVIEW

**Review by claudiosanches — CHANGES_REQUESTED** [review]
It doesn't seems to match with CLDR list: https://github.com/unicode-org/cldr/blob/release-36-1/common/subdivisions/en.xml#L1842-L1854

There's some different states.

**Comment by tolu-paystack** [comment]
@claudiosanches I got the list of regions from here https://en.wikipedia.org/wiki/ISO_3166-2:GH. There was a region that was removed and 7 new ones added in November of last year, I think the CLDR list may not have been updated to reflect this

**Comment by claudiosanches** [comment]
@tolu-paystack here a list released 7 days ago with the new regions: https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L1845-L1864
As I said, we don't use ISO anymore, also CLDR.

File: i18n/states.php [hunk 1]
```diff
@@ -349,6 +349,24 @@
 	),
 	'FI' => array(),
 	'FR' => array(),
+	'GH' => array(
+		'AF' => __('Ahafo', 'woocommerce'),
```
Comment (claudiosanches) [inline i18n/states.php:353]: Also need to fix coding standards.

**Review by claudiosanches — CHANGES_REQUESTED** [review]

File: i18n/states.php (hunk 1 above)
Comment (tolu-paystack) [reply i18n/states.php]: @claudiosanches done

File: i18n/states.php [hunk 2]
```diff
@@ -349,6 +349,24 @@
 	),
 	'FI' => array(),
 	'FR' => array(),
+	'GH' => array( // Ghanaian Regions
```
Comment (claudiosanches) [inline i18n/states.php:352]: Seems like that is missing two regions listed on https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L1845-L1864

**Review by claudiosanches — CHANGES_REQUESTED** [review]

File: i18n/states.php (hunk 2 above)
Comment (tolu-paystack) [reply i18n/states.php]: Sorry @claudiosanches I missed your earlier comment about the updated CLDR list. I've added the two regions

**Comment by tolu-paystack** [comment]
Hey @claudiosanches is this ok now?

**Review by claudiosanches — APPROVED** [review]
Thanks for your contribution!

**Comment by webaware** [comment]
Somehow CLDR has incorporated a municipality of Greenland in its list of provinces of Ghana. [Avannaata Kommunia](https://www.iso.org/obp/ui/#iso:code:3166:GL) should be removed from regions of Ghana. [The CLDR code indicates Greenland](https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L1864), so the error is evident even there.

**Comment by claudiosanches** [comment]
@webaware thank you very much for catching this bug, I made a fix for it in https://github.com/woocommerce/woocommerce/pull/26639

**Pull request merged**